.travis.yml

```diff
@@ -47,5 +52,6 @@ deploy:
     file: "netdata*.tar.*"
     on:
       condition: $CC = gcc
+      condition: $TRAVIS_OS_NAME = linux
```

Comment:
```
if [[ $CC = gcc && $TRAVIS_OS_NAME = linux ]]; then echo test; fi
```
works on the command line. I'll fix condition. Thank you for finding the error now!